js/mviewerstudio.js

```diff
@@ -909,3 +909,20 @@ $(".checkedurl").change(mv.checkURL);
 $("#mod-importfile").on('shown.bs.modal', function () {
     mv.getListeApplications();
 });
+
+
+var uploadSldFileToBackend = function(e) {
+    var sldFile = e.files[0];
+    $.ajax('/srv/store/style', {
+        data: sldFile,
+        method: 'POST',
+        processData: false,
+        success: function(data) {
+            $("#frm-sld").val(data.filepath)
+        },
+        error: function() {
+            alert("Le fichier n'a pas pu être chargé")
```

Comment:
Needs i18n like this ? 
https://github.com/geobretagne/mviewerstudio/blob/052a667decb8f714ea62a90ac2a21ace58b1d043/js/mviewerstudio.js#L773